pages/components-playground.tsx

```diff
@@ -1,14 +1,30 @@
 import {
+  BookmarkButton,
   Button,
   CommentButton,
-  LikeButton
+  LikeButton,
+  ShareButton
 } from "components/button";
 import Headline from "components/Headline"
 import CustomImageComponent from "components/CustomImageComponent"
 
 function Playground() {
   return (
     <>
+      <Headline>
+        This is a Headline for da playground
+      </Headline>
+      <ShareButton
+      ></ShareButton>
+      <BookmarkButton
+      ></BookmarkButton>
```

Comment:
By axing the children props, we should be able to just render these components like so
```
<ShareButton/>
<BookmarkButton/>
```